src/pages/subreddit/Subreddit.js

```diff
@@ -0,0 +1,46 @@
+import React, {useEffect, useState} from 'react';
+import {useParams} from "react-router-dom";
+import axios from "axios";
+
+function Subreddit() {
+    const { id } = useParams();
+
+    const [reddits, setReddits] = useState({});
```

Comment:
Ik zou de default waarde `undefined` maken, zodat je makkelijk kan checken of je een geldig object in handen hebt om iets mee te doen of niet:
```suggestion
    const [reddits, setReddits] = useState(undefined);
```